starboard/shared/modular/starboard_layer_posix_stat_abi_wrappers.cc

```diff
@@ -23,13 +23,21 @@ int stat_helper(int retval,
 
   musl_info->st_size = stat_info->st_size;
   musl_info->st_mode = stat_info->st_mode;
+
   musl_info->st_atim.tv_sec = stat_info->st_atim.tv_sec;
   musl_info->st_atim.tv_nsec = stat_info->st_atim.tv_nsec;
   musl_info->st_mtim.tv_sec = stat_info->st_mtim.tv_sec;
   musl_info->st_mtim.tv_nsec = stat_info->st_mtim.tv_nsec;
   musl_info->st_ctim.tv_sec = stat_info->st_ctim.tv_sec;
   musl_info->st_ctim.tv_nsec = stat_info->st_ctim.tv_nsec;
 
+  musl_info->st_nlink = stat_info->st_nlink;
+  musl_info->st_uid = stat_info->st_uid;
+  musl_info->st_gid = stat_info->st_gid;
+  musl_info->st_ino = stat_info->st_ino;
+  musl_info->st_dev = stat_info->st_dev;
+  musl_info->st_rdev = stat_info->st_rdev;
```

Comment:
Let's copy all of the fields over (this is missing st_blksize and st_blocks), I ran into a pretty nasty error with some fields not being copied over before